src/main/java/seedu/address/logic/commands/sort/SortInternshipsCommand.java

```diff
@@ -13,17 +13,13 @@
  */
 public class SortInternshipsCommand extends SortCommand {
 
-    // TODO: more defensive
     private final Comparator<Internship> sortComparator;
 
     public SortInternshipsCommand(String sortOrder, boolean reverse) {
-        Comparator<Internship> baseComparator = (int1, int2) -> {
-            if (sortOrder.equalsIgnoreCase("name")) {
-                return int1.getName().compareTo(int2.getName());
-            } else {
-                return int1.getFrom().compareTo(int2.getFrom());
-            }
-        };
+        Comparator<Internship> baseComparator =
+                sortOrder.equalsIgnoreCase("name")
+                        ? Comparator.comparing(Internship::getName)
+                        : Comparator.comparing(Internship::getFrom);
```

Comment:
do we need to check for invalid sortOrder here? eg. not "name" and also not "time"? or can we assume that parser already handles this?

Reply:
The parser would have handled it.

Reply:
I think in parser it was quite safe so sortComparator doesn't have to check iirc, @wardetu please reassure

Reply:
Oh oops you did reassure lol mb my wifi